packages/mdc-line-ripple/foundation.js

```diff
@@ -77,10 +77,7 @@ class MDCLineRippleFoundation extends MDCFoundation {
    * @param {!number} xCoordinate
    */
   setRippleCenter(xCoordinate) {
-    const attributeString =
-        `transform-origin: ${xCoordinate}px center`;
-
-    this.adapter_.setAttr('style', attributeString);
+    this.adapter_.setStyle('transform-origin', `${xCoordinate}px center`);
```

Comment:
Does this not need to be `transformOrigin` to work with `element.style[propertyName]`?

Reply:
Nevermind, I guess either one works, even in IE 11.